sssd_test_framework/utils/authentication.py

```diff
@@ -188,16 +188,16 @@ def __init__(self, host: MultihostHost, fs: LinuxFileSystem) -> None:
         super().__init__(host)
         self.fs: LinuxFileSystem = fs
 
-    def password(self, username: str, password: str) -> bool:
+    def password_with_output(self, username: str, password: str) -> tuple[int, str, str]:
```

Comment:
returns tuple[int, int, str, str] same in docstring

Reply:
Seems like you are missing the latest changes from ssh password?

Reply:
@pbrezina I tried it with the latest changes, but there was an index error and the code failed to work. SU is less complicated than SSH and I don't think it's necessary to have the command exit codes included in this function. Having just the tuple[int, str, str] I think it's fine?